static/js/server_events_dispatch.js

```diff
@@ -128,10 +128,11 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             page_params['realm_' + event.property] = event.value;
             realm_settings[event.property]();
             settings_org.sync_realm_settings(event.property);
-            if (event.property === 'create_stream_by_admins_only') {
+            if (event.property === 'create_stream_policy') {
                 if (!page_params.is_admin) {
+                    // TODO: Add waiting_period_threshold logic here.
```

Comment:
I also wasn't sure if it was alright to leave this as a to-do in the same way as the invite to stream policy.